src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -167,6 +167,13 @@ void AddSymbolCompletions(IDictionary<string, CompletionItem> result, IEnumerabl
             {
                 foreach (var function in @namespace.Descendants.OfType<FunctionSymbol>())
                 {
+                    if (function.FunctionFlags.HasFlag(FunctionFlags.ParamDefaultsOnly) && !(enclosingDeclarationSymbol is ParameterSymbol))
```

Comment:
Just out of curiosity, what'd be an example of a function with paramDefaultsOnly flag that is not bound to a parameter symbol?

Reply:
There aren't any.

Out of all the functions if we see one with the flag and the current position is not inside a parameter, we skip suggesting it. In other words, we show the functions with the flag only if current position is inside a param.